spring-boot-modules/pom.xml

```diff
@@ -63,10 +63,6 @@
         <module>spring-boot-nashorn</module>
         <module>spring-boot-parent</module>
         <module>spring-boot-performance</module>
```

Comment:
Having removed these from spring-boot-modules, I don't see how they'll be included in our parent pom and be able to build with jdk9+ profiles.
I guess we'll have to declare them explicitly there.